src/renderer/qssWidget/js/qssMouseWidget.js

```diff
@@ -0,0 +1,230 @@
+                    },
+                    components: {
+                        indicators: {
+                            type: "gpii.qssWidget.baseStepper.indicators",
+                            container: "{mouse}.dom.indicators",
+                            options: {
+                                model: {
+                                    setting: "{mouseSpeed}.model.setting"
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            swapMouseButtons: {
+                type: "gpii.qssWidget.mouseWidgetToggle",
+                container: "{that}.dom.swapMouseButtons",
+                options: {
+                    model: {
+                        setting: "{mouse}.model.setting.settings.swapMouseButtons"
+                    }
+                }
+            },
+            easierDoubleClick: {
+                type: "gpii.qssWidget.mouseWidgetToggle",
+                container: "{that}.dom.easierDoubleClick",
+                options: {
+                    model: {
+                        setting: "{mouse}.model.setting.settings.easierDoubleClick"
+                    }
+                }
+            },
+            largerMousePointer: {
+                type: "gpii.qssWidget.mouseWidgetToggle",
+                container: "{that}.dom.largerMousePointer",
+                options: {
+                    model: {
+                        setting: "{mouse}.model.setting.settings.largerMousePointer"
+                    }
+                }
+            },
+            channelNotifier: {
+                type: "gpii.psp.channelNotifier",
+                options: {
+                    events: {
+                        // Add events the main process to be notified for
+                        onQssWidgetSettingAltered:       "{mouse}.events.onQssWidgetSettingAltered",
+                        onQssWidgetNotificationRequired: "{mouse}.events.onQssWidgetNotificationRequired"
+                    }
+                }
+            }
+        }
+    });
+
+    /**
+     * Represents the QSS mouse toggle widget.
+     */
+    fluid.defaults("gpii.qssWidget.mouseWidgetToggle", {
+        gradeNames: ["fluid.viewComponent", "gpii.psp.selectorsTextRenderer"],
+
+        selectors: {
+            toggleButton: ".flc-toggleButton",
+            settingTitle: ".flc-qssMouseWidgetToggle-settingTitle"
+        },
+
+        enableRichText: true,
+
+        model: {
+            setting: {},
+            value: "{that}.model.setting.value",
+            messages: {
+                settingTitle: "{that}.model.setting.schema.title"
+            }
+        },
+
+        components: {
+            toggleButton: {
+                type: "gpii.psp.widgets.switch",
+                container: "{that}.dom.toggleButton",
+                options: {
+                    model: {
+                        enabled: {
```

Comment:
Having an expander which reads model values on startup is liable to races depending on when the source component gets initialised, and doesn't take care of keeping the values synchronised subsequently. It looks like this application would best be handled using the valueMapper transform, of which there is a short example in the docs: https://docs.fluidproject.org/infusion/development/ModelTransformationAPI.html#example-1-shorthand-with-primitive-output-values